src/main/java/bio/terra/service/dataset/flight/ingest/IngestCleanupStep.java

```diff
@@ -10,7 +10,7 @@
 import org.slf4j.LoggerFactory;
 
 public class IngestCleanupStep implements Step {
-    private Logger logger = LoggerFactory.getLogger("bio.terra.service.dataset.ingest");
+    private Logger logger = LoggerFactory.getLogger("bio.terra.service.dataset.flight.ingest");
```

Comment:
The _cool kids_ way to get the logger is:
```
private final Logger logger = LoggerFactory.getLogger(IngestCleanupStep.class);
```
No need to change it here, but it is our intended standard. We have a ticket to fix them.